test/gie/builtins.gie

```diff
@@ -2194,23 +2194,23 @@ operation +proj=igh_o   +a=6400000
 -------------------------------------------------------------------------------
 tolerance 0.1 mm
 accept  2 1
-expect  222687.614453779 111701.072127637
+expect  223538.245169845 111701.072127637
```

Comment:
It would be useful to test roundtripping on those points by adding `roundtrip 1` after each expect, like:

```suggestion
expect  223538.245169845 111701.072127637
roundtrip 1
```
Actually the tests should test that for each of the 12 zones rather than on those dummy standard points.

Reply:
> Actually the tests should test that for each of the 12 zones rather than on those dummy standard points.

This is a key point to making sure this works as expected. Please adjust the test accordingly. That is, add accept/expect coordinate pairs that fall within each of the twelve zones.

Reply:
I added in sub-projection tests for the new igh_o projection along with the existing igh projection.

Reply:
Great! Even's comment about adding `roundtrip 1`'s after each `expect` line is still valid. Apart from that I'd say this PR is good to go.